modules/http_push.c

```diff
@@ -22,6 +22,7 @@
 
 #include <syslog.h>
 #include <curl.h>
+#include <openssl/hmac.h>
```

Comment:
Is there some smaller HMAC implementation available? We don't want to depend on OpenSSL.